cmd/root/config/connection-strings.go

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+package config
+
+import (
+	"fmt"
+	"github.com/microsoft/go-sqlcmd/internal/cmdparser"
+	"github.com/microsoft/go-sqlcmd/internal/config"
+	"github.com/microsoft/go-sqlcmd/internal/output"
+	"github.com/microsoft/go-sqlcmd/internal/pal"
+	"github.com/microsoft/go-sqlcmd/internal/secret"
+)
+
+type ConnectionStrings struct {
+	cmdparser.Cmd
+}
+
+func (c *ConnectionStrings) DefineCommand(...cmdparser.Command) {
+	c.Cmd.Options = cmdparser.Options{
+		Use:   "connection-strings",
+		Short: "Display connections strings for the current context",
+		Examples: []cmdparser.ExampleInfo{
+			{
+				Description: "List connection strings for all client drivers",
+				Steps: []string{
+					"sqlcmd config connection-strings",
+					"sqlcmd config cs"},
+			},
+		},
+		Run:     c.run,
+		Aliases: []string{"cs"},
+	}
+
+	c.Cmd.DefineCommand()
+}
+
+func (c *ConnectionStrings) run() {
+	// connectionStringFormats borrowed from "portal.azure.com" "connection strings" pane
+	var connectionStringFormats = map[string]string{
+		"ADO.NET": "Server=tcp:%s,%d;Initial Catalog=%s;Persist Security Options=False;User ID=%s;Password=%s;MultipleActiveResultSets=False;Encode=True;TrustServerCertificate=False;Connection Timeout=30;",
```

Comment:
>Encode=True [](http://example.com/codeflow?start=135&length=11)

what is "Encode" ?

Reply:
Added more general issue to review these connection strings:

"Review sqlcmd config connection-strings"

[https://github.com/microsoft/go-sqlcmd/issues/170](https://github.com/microsoft/go-sqlcmd/issues/170)